### PR TITLE
expo instructions

### PR DESCRIPTION
**Pull request by mfmurray**

### CODE REVIEW

File: sdk-reference/mobile/react-native.mdx [hunk 1]
```diff
@@ -15,6 +15,74 @@ First, install the Rownd SDK for React Native.
 npm install @rownd/react-native
 ```
 
+### Expo development
+
+1. Add `@rownd/native` as a plugin to your `app.json` file.
```
Comment (mhamann) [inline sdk-reference/mobile/react-native.mdx:20]: This should be `@rownd/react-native`, right?

File: sdk-reference/mobile/react-native.mdx [hunk 2]
```diff
@@ -15,6 +15,74 @@ First, install the Rownd SDK for React Native.
...
+            "deploymentTarget": "14.0"
+          }
+        }
+      ]
+    ]
+  }
+}
+```
+
+4. Install Sodium (iOS Framework). [Step 2 for iOS](#ios)
```
Comment (mhamann) [inline sdk-reference/mobile/react-native.mdx:58]: Is this possible without "ejecting" in an Expo project? Is the podfile available to edit without it getting overwritten?

File: sdk-reference/mobile/react-native.mdx (hunk 1 above)
Comment (mfmurray) [reply sdk-reference/mobile/react-native.mdx]: Wow just seeing these comments. Updated

File: sdk-reference/mobile/react-native.mdx (hunk 2 above)
Comment (mfmurray) [reply sdk-reference/mobile/react-native.mdx:58]: As far as I understand the @rownd/react-native SDK only works when "ejected". It also doesn't look like we could use an expo plugin to modify the podfile: https://docs.expo.dev/config-plugins/development-and-debugging/#modify-ios-podfile

**Review by BobbyRadford — APPROVED** [review]

**Pull request merged**